integration/apps/sinatra2-modular/app/parent.rb

```diff
@@ -0,0 +1,7 @@
+require 'sinatra/base'
+require 'ddtrace'
+require 'request_store'
+
+class Parent < Sinatra::Base
+  use ::RequestStore::Middleware
```

Comment:
I see that this gem was added, and we enable it here, but ... is it me or did we never end up using it at all?

Can we perhaps remove it, so as to simplify the example app?